core/storage/blog/gae_models.py

```diff
@@ -496,3 +507,144 @@ def create(cls, blog_post_id: str, author_id: str) -> BlogPostRightsModel:
         entity.put()
 
         return entity
+
+
+class BlogAuthorDetailsModel(base_models.BaseModel):
+    """Model for storing user's blog author details.
+
+    The id/key of instances of this model is randomly generated string of
+    length 12."""
+
+    # We use the model id as a key in the Takeout dict.
+    ID_IS_USED_AS_TAKEOUT_KEY: Literal[True] = True
+
+    author_id = datastore_services.StringProperty(indexed=True, required=True)
```

Comment:
This should have a comment above it: The user ID of the blog author.

Reply:
Added. Thanks